phaser/phase.py

```diff
@@ -403,6 +404,15 @@ def execute_batch_step(self, step):
         except Exception as exc:
             self.process_exception(exc, step, None)
 
+    def execute_context_step(self, step):
+        self.context.current_row = 'context'
+        try:
+            step(self.context)
+        except DropRowException as dre:
+            raise PhaserException("DropRowException can't be handled in a context_step") from dre
```

Comment:
Will the `PhaserException` raised here be caught and handled in the next line and sent to `self.process_exception`?